output/openai/articles/summaries/article-20251211-update-to-gpt-5-system-card-gpt-5-2.md

```
@@ -0,0 +1,28 @@
+
+本システムカードでは、GPT-5.2 Instantを「gpt-5.2-instant」と、GPT-5.2 Thinkingを「gpt-5.2-thinking」と呼称しています。
+
+## Key Points
+- GPT-5.2はGPT-5シリーズの最新モデルファミリーです。
+- GPT-5.2モデルの安全対策アプローチは、GPT-5およびGPT-5.1システムカードに記述されたものと同様に包括的です。
+- これらのモデルは、インターネット上の公開情報、サードパーティ情報、ユーザーや研究者が提供する情報など、多様なデータで訓練されています。
+- GPT-5.2 Instantはgpt-5.2-instant、GPT-5.2 Thinkingはgpt-5.2-thinkingとして言及されます。
+
+---
+
+*Generated by Provider News Monitor using Gemini 2.5 Flash*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)